controllers/operandrequest/reconcile_operand.go

```diff
@@ -200,69 +200,57 @@ func (r *Reconciler) reconcileCRwithConfig(ctx context.Context, service *operato
 }
 
 // reconcileCRwithRequest merge and create custom resource base on OperandRequest and CSV alm-examples
-func (r *Reconciler) reconcileCRwithRequest(ctx context.Context, requestInstance *operatorv1alpha1.OperandRequest, operand operatorv1alpha1.Operand, requestKey types.NamespacedName, csv *olmv1alpha1.ClusterServiceVersion) error {
-	almExamples := csv.GetAnnotations()["alm-examples"]
```

Comment:
So are we no longer using alm-examples to create CR? Does this mean we cannot use the default CR provided by operators' CSVs?

Reply:
This is only for creating CR by OperandRequest. We still support creating CRs by OperandConfig